client/blocks/reader-combined-card/index.jsx

```diff
@@ -176,8 +176,14 @@ function combinedCardPostKeyToKeys( postKey ) {
 export default connect( ( state, ownProps ) => {
 	const postKeys = combinedCardPostKeyToKeys( ownProps.postKey );
 
+	// Posts are directly supplied in Devdocs
```

Comment:
this comment seems like a strange coupling into the component back to the devdocs.

question one is why do the devdocs depend on this connected component? is it not easy to mock out the values that `connect()` normally provides?

question two is couldn't we remove this coupling by asserting a general behavior for the component…

> This component will prefer posts passed into it as props over those it extracts from state.

or

> This component will search for posts in state if none are provided via `props`

Reply:
The second explanation reads great to me. I agree that highlighting just Devdocs as a place where props are mocked is too tightly coupled as this kind of prop mocking/injection is handy for testing too.

You could always say "for example, `posts` fixtures are used in devdocs". But I don't even think it's needed; this sort of mocking behaviour reads as standard practice at least to me.

Reply:
👍 I'm going to take your advice and use the un-`connect`-ed component in Devdocs.